integration/localnet/bootstrap.go

```diff
@@ -1,49 +1,58 @@
 package main
 
 import (
+	"encoding/hex"
 	"encoding/json"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"runtime"
 	"time"
 
-	"github.com/plus3it/gorecurcopy"
-	"gopkg.in/yaml.v2"
-
+	"github.com/go-yaml/yaml"
+	"github.com/plus3it/gorecurcopy
```

Comment:
missing `"`

Reply:
Sorry about that, very bad error. My mistake, I've corrected it.